flake.nix

```diff
@@ -58,14 +63,14 @@
 
             buildPhase = ''
               cd ..
+              export PYTHONPATH="$PWD:$PYTHONPATH"
```

Comment:
Maybe by getting rid of the `diffkemp` import in `simpll_build.py` file we could get also rid of this?

Reply:
I think that we could, yes.